engine/src/execution_context.rs

```diff
@@ -129,7 +129,7 @@ impl<'e, U> ExecutionContext<'e, U> {
     }
 
     #[inline]
-    pub(crate) fn get_field_value_unchecked(&self, field: Field<'_>) -> &LhsValue<'_> {
+    pub(crate) fn get_field_value_unchecked(&self, field: &Field) -> &LhsValue<'_> {
```

Comment:
Should this take a `FieldRef` instead of a `&Field` (and similarly for `get_list_matcher_unchecked` below)?

Reply:
I don't think so, since this is used specifically to get the value from a Field stored in the ast, it seems wasteful to have to convert to a FieldRef before.